internal/wrapnix/wrapper.go

```diff
@@ -33,7 +34,7 @@ var wrapper string
 var wrapperTemplate = template.Must(template.New("wrapper").Parse(wrapper))
 
 // CreateWrappers creates wrappers for all the executables in nix paths
-func CreateWrappers(ctx context.Context, devbox devboxer) error {
+func CreateWrappers(ctx context.Context, devbox devboxer, pure bool) error {
```

Comment:
Adding `pure` here makes this PR quite a bit more complicated because booleans need to be threaded in more places. It's also not really a "wrapper", but I see why creating it here is simpler.

To keep changes simpler I would suggest always creating the devbox symlink (so get rid of `pure` here and just assume it's always true). There's no downside to having the symlink exist in non-pure environments.